### DISCUSSION TITLE
Ignore coverages on imports

### DISCUSSION THREAD

**Discussion by junioralcant** [vitest-dev/vitest#3959] [Q&A]
### Describe the bug

Does anyone know how I can solve this problem? He is covering the imports.

![Captura de Tela 2023-08-07 às 09 51 35](https://github.com/vitest-dev/vitest/assets/40610119/eb8f3fcb-aba3-4ab1-8dd7-7452df80d9e5)

### Reproduction

N/D 

### System Info

```shell
System:
    OS: macOS 12.5.1
    CPU: (8) arm64 Apple M1
    Memory: 88.28 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.15.0 - /opt/homebrew/opt/node@16/bin/node
    Yarn: 1.22.17 - /opt/homebrew/bin/yarn
    npm: 8.5.5 - /opt/homebrew/opt/node@16/bin/npm
    Watchman: 2022.07.04.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 115.0.5790.170
    Safari: 15.6.1
  npmPackages:
    @vitejs/plugin-react: ~4.0.0-beta.0 => 4.0.4 
    @vitest/coverage-c8: ~0.32.4 => 0.32.4 
    @vitest/coverage-istanbul: ^0.33.0 => 0.33.0 
    vite: ~4.3.0 => 4.3.9 
    vitest: ~0.30.1 => 0.30.1
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by AriPerkkio** [comment]
The coverage report doesn't look right. If you could provide a minimal reproduction case for this I can take a look at this. 

Generally, the `@vitest/coverage-c8` and `@vitest/coverage-v8` will mark `import` statements as covered. You can use `@vitest/coverage-istanbul` if this is an issue.
